rebar.config.script

```diff
@@ -20,6 +20,19 @@ FilterDepsRocksDb = fun(Deps) ->
                         end
                     end,
 
+%% In order to avoid ugly dynamic module loading over rpc - inject meck into the common test release
+%% This does not inject any applications into the system tests, smoke tests and production releases
+Args = init:get_plain_arguments(),
+IsCommonTestRelease = lists:member("test", Args) and lists:member("release", Args),
+AddMeckInCommonTests = fun(Apps) ->
```

Comment:
This seems overly complicated. Though I admit I did not check this is detail.

It seems to me though that it is a generic test infrastructure infrastructure that can be merged indipendently from the feature: if so, may I suggest splitting this infrastructure as separate PR against master?

(I suspect this PR will stay open for a while and I am concerned of re-rebases, and of re-reviews. Hence my advice.)